controllers/ibmblockcsi_controller.go

```diff
@@ -193,6 +194,11 @@ func (r *IBMBlockCSIReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		return reconcile.Result{}, err
 	}
 
+	callHomeSyncer := clustersyncer.NewCallHomeSyncer(r.Client, r.Scheme, instance)
+	if err := syncer.Sync(context.TODO(), callHomeSyncer, r.Recorder); err != nil {
+		return reconcile.Result{}, err
+	}
```

Comment:
what happens if call home is disabled? is here where we avoid creating the cronjob?
do we always return here? if so, does it prevent the rest of the driver from being synced?

Reply:
now it will delete the call home Cronjob if it exists but is erased from CR or secret removed. anyway, it won't interrupt the operator running and syncing.